pkg/pod/container.go

```diff
@@ -235,6 +235,31 @@ func (builder *ContainerBuilder) WithEnvVar(name, value string) *ContainerBuilde
 	return builder
 }
 
+// WithVolumeMount  adds a pod volume mount inside the container.
+func (builder *ContainerBuilder) WithVolumeMount(volMount v1.VolumeMount) *ContainerBuilder {
+	glog.V(100).Infof("Adding VolumeMount to the %s container's definition", builder.definition.Name)
```

Comment:
Wondering if it helpful to log the volume name or mount path ? This could be moved after the Name and MountPath checks are done.

Based on the example in the comments, and the fact that the volume it will be created in the test, I would add some more details.

Reply:
Added logging of `VolumeMount`'s _Name_ and _Path_